web-share/src/js/service-worker.js

```diff
@@ -11,6 +11,15 @@ import {mimeRoute as videosRoute} from '../svelte/routes/Videos.svelte';
 
 const broadcastChannel = 'BroadcastChannel' in self ? new BroadcastChannel(channelName) : null;
 
+self.addEventListener('contentdelete', async (event) => {
```

Comment:
Maybe a comment explaining that this event is fired when a user (or browser) has deleted some content?

Reply:
Done.